interfaces/backends/backends_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	. "gopkg.in/check.v1"
 
 	"github.com/snapcore/snapd/interfaces/backends"
-	"github.com/snapcore/snapd/release"
+	"github.com/snapcore/snapd/sandbox/apparmor"
```

Comment:
(nitpick) Given that pretty much everything else imports this as apparmor_sandbox I wonder if we should do it here too?